### PR TITLE
Release notes for 6.5.2

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: modules/release-notes/pages/relnotes.adoc [hunk 1]
```diff
@@ -5,6 +5,143 @@ This release seeks to improve developer agility, reduce total cost of ownership,
...
+| *Summary*: When making the debug log async, the DCP client log level was incorrectly set to ERROR. Fixed this issue to use the global service log level instead, which defaults to DEBUG.
+|===
+
+==== Cluster Manager
+
+[#table_fixedissues_v652-cluster-manager,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-42607[MB-42607^]
```
Comment (daverigby) [inline modules/release-notes/pages/relnotes.adoc:44]: This MB should be moved to data service section.

File: modules/release-notes/pages/relnotes.adoc [hunk 2]
```diff
@@ -5,6 +5,143 @@ This release seeks to improve developer agility, reduce total cost of ownership,
...
+| *Summary*: During TLS handshake with the Data Service, if the node certificate required more than 8192 bytes to transmit, then the handshake failed. This has been fixed.
+|===
+
+==== Data Service
+
+[#table_fixedissues_v652-data,cols="25,66"]
+|===
+| Issue | Description
```
Comment (daverigby) [inline modules/release-notes/pages/relnotes.adoc]: As an aside, is there any particular order to the MBs listed here? If not, I'd suggest sorting by number to aid the customer in finding things (and also to make it easier to review them!)

File: modules/release-notes/pages/relnotes.adoc [hunk 3]
```diff
@@ -5,6 +5,143 @@ This release seeks to improve developer agility, reduce total cost of ownership,
...
+
+| https://issues.couchbase.com/browse/MB-40543[MB-40543^]
+| *Summary*: Added the ability to re-enable FTS logging in memcached for debug purposes.
+
+| https://issues.couchbase.com/browse/MB-40531[MB-40531^]
+| *Summary*: Fixed a performance regression where the HiFi-MRU Item Pager (added in 5.0) would evict more active items compared to previous LRU ItemPager under certain workloads.
+
+| https://issues.couchbase.com/browse/MB-38283[MB-38283^]
+| *Summary*: Fixed an issue where Memcached log wrapped due to excessive DCP "no memory" messages.
+|===
```
Comment (daverigby) [inline modules/release-notes/pages/relnotes.adoc:72]: Also need MB-41300 adding. I've asked Paolo to add a comment to that MB with details.

Comment (daverigby) [inline modules/release-notes/pages/relnotes.adoc:72]: And MB-41407 adding. I've asked James to add a comment to the MB with the details.

Comment (daverigby) [inline modules/release-notes/pages/relnotes.adoc:72]: And MB-41942 adding, asked Ben to add a comment for it.

Comment (daverigby) [inline modules/release-notes/pages/relnotes.adoc:72]: ... also need MB-42610. Ben to add comment to MB

Comment (daverigby) [inline modules/release-notes/pages/relnotes.adoc:72]: ... and MB-42918. Again, Ben to add a comment to MB for it.

**Review by daverigby — CHANGES_REQUESTED** [review]

File: modules/release-notes/pages/relnotes.adoc (hunk 2 above)
Comment (amarantha-k) [reply modules/release-notes/pages/relnotes.adoc]: Typically, we group MBs by component and within each group, they are sorted in descending order.

Comment (daverigby) [reply modules/release-notes/pages/relnotes.adoc]: Ah yes, sorry I thought they were not in numerical order but I think I was looking across different tables :)

File: modules/release-notes/pages/relnotes.adoc (hunk 3 above)
Comment (amarantha-k) [reply modules/release-notes/pages/relnotes.adoc:72]: MB-41942 is private and needs to be made public before we can publish in release notes.

**Review by mihirkamdarqe — COMMENTED** [review]
The links for Indexing bugs MB-43041 & MB-43046 are incorrect.

**Review by arod1987 — APPROVED** [review]

**Pull request merged**